OrbitCore/ConnectionManager.cpp

```diff
@@ -73,6 +75,33 @@ void ConnectionManager::InitAsRemote()
     m_Thread = std::make_unique<std::thread>(&ConnectionManager::RemoteThread, this);
 }
 
+//-----------------------------------------------------------------------------
+void ConnectionManager::SetSelectedFunctionsOnRemote( const char* a_Data, size_t a_Size ) {
```

Comment:
This should either take in a Message or a string as argument, it would be clearer than passing in a raw buffer.